code/modules/holiday/spacemas.dm

```diff
@@ -763,11 +763,6 @@ proc/compare_ornament_score(list/a, list/b)
 
 // Santa Stuff
 
-/obj/item/card/id/captains_spare/santa
```

Comment:
if you do this, please make some alternative all-access card that can be invoked with `spawn re/sa` or similar for adminning/devving purposes